mr_module/TimeMachine/TimeMachineController.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace MrModule\TimeMachine;
+
+use Mr\Http\Controllers\Controller;
+use MrModule\TimeMachine\TimeMachine;
+
+class TimeMachineController extends Controller
+{
+    public function index() {
+        $results = TimeMachine::all();
```

Comment:
I know i use ::all() for a lot of index() methods. In future i will have to change them to support paging, filter, and sorting.